src/routes/view-content/+page.svelte

```diff
@@ -354,7 +354,7 @@
                 {:else if currentBible?.loadingContent}
                     <FullPageSpinner />
                 {:else if currentBible?.content?.chapters?.length}
-                    <div class="flex-start prose w-full overflow-y-scroll">
+                    <div class="flex-start prose mx-auto w-full overflow-y-scroll">
```

Comment:
This makes sure the Bible is centered on larger screens.

Reply:
I removed mx auto because when you selected one short verse, it would be in the center. I didn't think that was ideal. But I will admit it's an edge case.

Reply:
Oh gotcha! I will mess around with it to try to handle both cases.

Reply:
I just tested and it looks like one verse is showing up correctly with this approach. Maybe it was the placement of the mx-auto before? Should be good now.